utils/create_dummy_models.py

```diff
@@ -504,6 +504,27 @@ def convert_feature_extractor(feature_extractor, tiny_config):
     if to_convert:
         feature_extractor = feature_extractor.__class__(**kwargs)
 
+    # Sanity check: on tiny image feature extractors, a large image size results in slow CI -- up to the point where it
+    # can result in timeout issues.
+    if (
+        isinstance(feature_extractor, BaseImageProcessor)
+        and hasattr(feature_extractor, "size")
+        and isinstance(feature_extractor.size, dict)
+    ):
+        largest_image_size = max(feature_extractor.size["height"], feature_extractor.size["width"])
+        if largest_image_size > 64:
+            # hardcoded exceptions
+            models_with_large_image_size = ("deformable_detr", "flava", "grounding_dino", "mgp_str", "swiftformer")
+            if any(model_name in feature_extractor.__class__.__name__ for model_name in models_with_large_image_size):
```

Comment:
`feature_extractor.__class__.__name__` is the class name of a feature extractor, which is unlikely to contain model type names like `deformable_detr` or `grounding_dino`. Also the upper/lower cases should be considered.

Reply:
(or just use the class names in `models_with_large_image_size`)